README.md

```diff
@@ -6,7 +6,7 @@
 4> Import the committee table using committee.sql.</br>
 5> Open the acm-nit-surat/index.html in localhost server.
 
-## Things to take care while you create a PR
+## Things to take care while you create a Pull request.
```

Comment:
@YashGolechha, make 'R' capital in 'Pull request', merge the two commits into one and again create a PR !